lms/templates/navigation.html

```diff
@@ -76,6 +77,23 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
     </ol>
     <ol class="user">
       <li class="primary">
+          <form action="/i18n/setlang/" method="post" class="settings-language-form" id="language-settings-form">
```

Comment:
I'm thinking that there should be a configuration setting to specify whether or not to show the language selector. Is this a feature that we'll be enabling on edx.org? Even if it is, it isn't clear that all Open edX sites will want this.

I also would only show the selector if there is more than one released language. Having a selector with only a single item is unnecessary.